### DISCUSSION TITLE
less compile diffrence at inline-style or .css?

### DISCUSSION THREAD

**Discussion by xjccc** [nuxt/nuxt#16172] [Questions] [answered]
at inline-style color: rgba is right
at .css file color: rgba -> argb?
![A94A092710C25EA945A4F2876B1C50E2](https://user-images.githubusercontent.com/16478776/206667091-431adfa3-370c-495a-8de2-02c4ab2825b1.jpg)

**Comment by TheAlexLichter** [comment, marked as answer]
**Udpdate:** Okay. This is related to vite - see the [vite docs on buildOptions - `cssTarget`](https://vitejs.dev/config/build-options.html#build-csstarget)

---

Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Reply by xjccc** [reply]
> Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

https://stackblitz.com/edit/github-fb3w7h?file=app.vue
npm run build
in node-server/_nuxt/entry.css  the color is #03102a99

**Reply by TheAlexLichter** [reply]
Okay. This is related to vite - see the [vite docs on buildOptions - `cssTarget`](https://vitejs.dev/config/build-options.html#build-csstarget)

**Reply by xjccc** [reply]
> Okay. This is related to vite - see the [vite docs on buildOptions - `cssTarget`](https://vitejs.dev/config/build-options.html#build-csstarget)

ye. thk u very much~
